### PR TITLE
Basic footer for doc-page - 10%

### LINKED ISSUE #21: Basic footer for doc-page - 10%
Footer for the doc page - we really don't have any link now - maybe just a dark area with a copyright statement at the center is enough for now ... easy task!

### PR DESCRIPTION
**Pull request by yuki-tylar**
Fixes #21

### CODE REVIEW

**Comment by pmvald** [comment]
Good! When you finish your PR use the review button the right to "request review" - this way we know that this PR is done and ready to review ... so, is this PR ready?

File: src/pages/__mocks/doc-page-res.tsx [hunk 1]
```diff
@@ -85,6 +85,9 @@ export default function () {
 					</div>
 				</div>
 			</div>
+			<div className="py-hq px-2 text-center" style={{backgroundColor: '#0d1117', color: '#e5e5e5', fontWeight: 300, fontSize: '14px'}}>
```
Comment (pmvald) [inline src/pages/__mocks/doc-page-res.tsx:88]: class `ss-doc-footer` maybe?

**Review by pmvald — CHANGES_REQUESTED** [review]

**Comment by yuki-tylar** [comment]
Yes it's ready to merge :) 

Sorry I can't find `review button the right to "request review"`. Could you please tell me where it is?

**Comment by pmvald** [comment]
I think you should "re-request review" - using the icon that is similar to refresh icon ...

**Comment by pmvald** [comment]
I think you should "re-request review" - using the icon that is similar to refresh icon ...

**Comment by yuki-tylar** [comment]
I tried re-request. I hope it works :)

**Comment by pmvald** [comment]
Yes it did! I'll review it soon

Sent from Yahoo Mail on Android 
 
  On Mon., 24 Jan. 2022 at 2:03 p.m., ***@***.***> wrote:   


I tried re-request. I hope it works :)

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Review by pmvald — APPROVED** [review]

**Pull request merged**